icu4c/APIChangeReport.md

```diff
@@ -0,0 +1,957 @@
+
```

Comment:
Please add a copyright header. See for example https://raw.githubusercontent.com/unicode-org/icu/master/docs/processes/fuzzer_targets.md

Reply:
done, will show up in next update

Reply:
(pushed to #807)